Software/scan_key/ErrorSQL.py

```diff
@@ -3,11 +3,19 @@
 import pymysql
 import os
 import smtplib
+import configparser
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
 def errorSQL(id, errMessage):
-    cnx = pymysql.connect(user='CATS', password='CATS', host='192.168.1.2', database='CATS', autocommit=True)
+	config = configparser.RawConfigParser() #instantiate config reader
```

Comment:
I agree with this, however I think it should just be made into a new file called sql.py or something and then we can have errorsql.py handle errors and authsql.py handle authentication or something.

Or have send and get methods in sql.py